.snyk

```diff
@@ -1,10 +1,11 @@
 # Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
 version: v1.25.0
 # ignores vulnerabilities until expiry date; change duration by modifying expiry date
-ignore:
-  SNYK-JS-BRACES-6838727:
-    - '*':
-        reason: no fix; no impact
-        expires: 2099-01-01T00:00:00.000Z
-        created: 2024-05-14T06:44:55.382Z
+ignore: {}
+  # FIXED by bumping to braces@3.0.3
+  # SNYK-JS-BRACES-6838727:
+  #   - '*':
+  #       reason: no fix; no impact
+  #       expires: 2099-01-01T00:00:00.000Z
+  #       created: 2024-05-14T06:44:55.382Z
```

Comment:
do we need this here as a comment?

Reply:
i left this in on submit to as a reference for future selves as it can be a pain to download the snyk cli and generate the file. for consistency with submit alone i would leave in for that reason